src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+(function() {
+    let translations = {
+            'Passwords'                           : 'Mots de passe',
+            'All'                                 : 'Tous',
+            'Folders'                             : 'Dossiers',
+            'Recent'                              : 'Récents',
+            'Favorites'                           : 'Favoris',
+            'Shared'                              : 'Partagés',
+            'Tags'                                : 'Mots clés',
+            'Security'                            : 'Sécurité',
+            'Trash'                               : 'Corbeille',
+            'Details'                             : 'Détails',
+            'Edit'                                : 'Modifier',
+            'Rename'                              : 'Renommer',
+            'Delete'                              : 'Effacer',
+            'Restore'                             : 'Restaurer',
+            'New Password'                        : 'Nouveau mots de passe',
+            'New Folder'                          : 'Nouveau dossier',
+            'New Tag'                             : 'Nouveau mot clé',
+            'Secure'                              : 'Sécurisé',
+            'Weak'                                : 'Faible',
+            'Weak (Duplicate)'                    : 'Faible (Dupliqué)',
+            'Weak (Outdated)'                     : 'Faible (Obsolète)',
+            'Breached'                            : 'Compromis',
+            'Notes'                               : 'Notes',
+            'Share'                               : 'Partager',
+            'Revisions'                           : 'Révisions',
+            'QR Code'                             : 'QR Code',
+            'Password'                            : 'Mot de passe',
+            'Username'                            : 'Nom d’utilisateur',
+            'Website'                             : 'Site internet',
+            'Add Tags...'                         : 'Ajouter des mots clés…',
+            'Color'                               : 'Couleur',
+            'Name'                                : 'Nom',
+            'Label'                               : 'Étiquette',
+            'General'                             : 'Général',
+            'Properties'                          : 'Propriétés',
+            'Numbers'                             : 'Nombre',
+            'Special Characters'                  : 'Caractères spéciaux',
+            'More Options'                        : 'Plus d’options',
+            'Favorite'                            : 'Favori',
+            'Encryption'                          : 'Chiffrement',
+            'On the server'                       : 'Sur le serveur',
+            'Custom Fields'                       : 'Champs Personnalisés',
```

Comment:
In French, you normally do not start in uppercase a word in the middle of a sentence unless it is a proper noun => `Champs personnalisés`